ksql-functional-tests/src/main/java/io/confluent/ksql/test/model/RecordNode.java

```diff
@@ -47,7 +47,7 @@
       @JsonProperty("window") final WindowData window
   ) {
     this.topicName = topicName == null ? "" : topicName;
-    this.key = key == null ? "" : key;
+    this.key = key;
```

Comment:
In case you're wondering...  a `null` key is value and so we shouldn't be converting to an empty string.  Without this change the new tests in `insert-values.json` would fail.